www/rustup.js

```diff
@@ -164,6 +164,33 @@ function fill_in_bug_report_values() {
     nav_app.textContent = navigator.appVersion;
 }
 
+function clear_copied_text() {
+    document.getElementById('copied-text').innerText = ''
+}
+
+function handle_copy_button_click() {
+    navigator.permissions.query({ name: 'clipboard-write' }).then(function(result) {
+        if (result.state === 'granted' || result.state === 'prompt') {
+            var rustupInstallCommand = new Blob(
+              ["curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh"],
+              { type: 'text/plain' }
+            );
+            var clippedItem = new ClipboardItem({ 'text/plain': rustupInstallCommand });
+
+            navigator.clipboard.write([clippedItem]).then(function() {
+                document.getElementById('copied-text').innerText = 'Copied!'
+            }, function(error) {
+                console.error('Attempt to write to clipboard failed. Error:');
+                console.dir(error);
+            });
+        } else {
+            console.log('clipboard-permission not granted:', result);
+        }
+    });
+
+    setTimeout(clear_copied_text, 2000)
```

Comment:
For a second, I thought we were somehow deleting the copied text itself (i.e. the text we just wrote to the clipboard). It would be nice if this function used a name which indicated clearly its usage of "this function is resetting the message that the user has copied the text." I'd also have it linger slightly longer.